<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cardinal样条曲线算法展示</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1013</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_grain">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>每个区间插值点个数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_grain">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_speed">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>小车速度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_speed">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>小车加速度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_acce">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_number">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>关键点个数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="number2">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>    0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_length">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>路线总长度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="length">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>    0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_tension">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>参数τ：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_tension">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_0">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_1">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="create">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>生成轨迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>显示插值点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始运动</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="stop_move">
           <property name="text">
            <string>暂停运动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="change">
           <property name="text">
            <string>更换小车</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="continue_move">
           <property name="text">
            <string>继续运动</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>清空窗口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pic_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
